src/components/Auth/api.test.ts

```diff
@@ -0,0 +1,144 @@
+import AuthApi from './api';
+import { AuthUser } from './types';
+
+describe('API', () => {
+  const hostAndPort = 'piroj.ok';
```

Comment:
Let's add a port just to make sure. While we're at it, please also change to a fictional host like `foo.example.com`.